packages/nextjs/pages/example-zk/SignedStats.tsx

```diff
@@ -0,0 +1,128 @@
+import type { NextPage } from "next";
+import { useBirthYearProofsStore } from "~~/services/store/birth-year-proofs";
+
+function shortenHashString(hash: string) {
```

Comment:
Should we move this two methods to a `utility` folder?

Reply:
In my opinion, if we manage to get this file a little bit smaller I think that we can keep it here so that it's easier to get an overview of what's happening.

Reply:
ill will do both, smaller file and move the functions to utils